pkg/alertmanager/amcfg.go

```diff
@@ -1210,6 +1210,7 @@ func (cb *configBuilder) convertTelegramConfig(ctx context.Context, in monitorin
 		VSendResolved:        in.SendResolved,
 		APIUrl:               in.APIURL,
 		ChatID:               in.ChatID,
+		MessageThreadID:      in.MessageThreadID,
```

Comment:
you'll need also to update the *telegramConfig.sanitize() function to drop the message thread id field if version < 0.26.0.